.github/workflows/test.yml

```
@@ -31,9 +31,17 @@ jobs:
           python -m pip install -U pip
           python -m pip install -U pytest
           python -m pip install .
+        env:
+          CFLAGS: '-DDEBUG'
 
       - name: Tests
         run: |
+          pytest -s
```

Let's use the long version of `-s` => `--capture=no` as it's not such a common option. Plus `python -m`:

```suggestion
          python -m pytest --capture=no
```